lib/gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -53,7 +54,7 @@ def self.in_depot_session(opts, &_block)
 
   def self.new_session(opts)
     klass = uri_scheme_to_class(opts[:uri])
-    session = klass.new(:uri => opts[:uri], :username => opts[:username], :password => opts[:password])
+    session = klass.new(:uri => opts[:uri], :username => opts[:username], :password => opts[:password], :region => opts[:region])
```

Comment:
Can we simplify this line to `session = klass.new(opts.slice(:uri. :username, :password, :region))`